app/src/main/java/org/fossasia/openevent/app/data/models/Attendee.java

```diff
@@ -93,34 +94,7 @@ public Attendee withCheckedIn() {
 
     @Override
     public int compareTo(@NonNull Attendee other) {
-        int compareFirstName;
-        int compareLastName;
-
-        // TODO: Reduce complexity and move logic outside the class
-        return
-            (compareFirstName = firstName
-                .toLowerCase()
-                .compareTo(
-                    other
-                        .getFirstName()
-                        .toLowerCase()
-                )
-            ) == 0 ? (compareLastName =
-                lastName
-                    .toLowerCase()
-                    .compareTo(
-                        other
-                            .getLastName()
-                            .toLowerCase()
-                    )
-            ) == 0 ?
-                email
-                    .toLowerCase()
-                    .compareTo(
-                        other
-                            .getEmail()
-                            .toLowerCase()
-                    ) : compareLastName : compareFirstName;
+        return getFirstName().compareTo(other.getFirstName());
```

Comment:
![screenshot_2017-07-20-13-10-52](https://user-images.githubusercontent.com/13533840/28406188-1e20655e-6d4d-11e7-9189-90b5f480e427.png)

with only firstname comparison `Amol Mejari` would appear on top of `amol lowercase`. Please keep the complete comparison as it is just a few lines of code, and not gonna run all the cases each time.